### PR TITLE
added current milk price to user page

### PR DESCRIPTION
**Pull request by MaximillianStewart**
added code to display milk price on the cows column

Before:
image

After:
image

### CODE REVIEW

**Review by MrJacobEisner — DISMISSED** [review]

**Review by Biggergig — DISMISSED** [review]
lgtm

**Review by pconrad — CHANGES_REQUESTED** [review]
Please do not remove .env.SAMPLE

image

**Comment by pconrad** [comment]
@MaximillianStewart Please consider adding screenshots and storybook links to the description, similar to #22

File: frontend/src/main/components/Commons/ManageCows.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
             {/* change $10 to info from fixture */}
             <Card.Title>Market Cow Price: ${commons?.cowPrice}</Card.Title>
             <Card.Title>Number of Cows: {userCommons.numOfCows}</Card.Title>
+            <Card.Title>Current Milk Price: ${commons?.milkPrice} </Card.Title>
```
Comment (pconrad) [inline frontend/src/main/components/Commons/ManageCows.js:15]: It appears the the storybook component for ManageCows needs to be updated to support this.  Right now it looks like this:

image

follow [link](https://ucsb-cs156-s23.github.io/proj-happycows-s23-6pm-3/prs/19/storybook/?path=/story/components-commons-managecows--uncontrolled) to see for yourself

I'll outline the fix in the next comment.

**Review by pconrad — CHANGES_REQUESTED** [review]
The storybook needs to be updated to support this change.

Here's how... in the file:

`frontend/src/stories/components/Commons/ManageCows.stories.mdx`

You'll see this line:

```jsx
    <ManageCows userCommons = {userCommonsFixtures.oneUserCommons[0]} onBuy={(userCommons) => { console.log("onBuy called:",userCommons); }} onSell={ (userCommons) => { console.log("onSell called:",userCommons); }} />
```

Which we can first reformat to make it easier to read:
```jsx
 <ManageCows 
    userCommons = {userCommonsFixtures.oneUserCommons[0]} 
    onBuy={(userCommons) => { console.log("onBuy called:",userCommons); }} 
    onSell={ (userCommons) => { console.log("onSell called:",userCommons); }} 
 />
```

We need to add another parameter `commons = {commonsFixtures.oneCommons[0]} ` to pass in a fixture for the commons, maybe something like this:

```
```jsx
 <ManageCows 
    commons = {commonsFixtures.oneCommons[0]} 
    userCommons = {userCommonsFixtures.oneUserCommons[0]} 
    onBuy={(userCommons) => { console.log("onBuy called:",userCommons); }} 
    onSell={ (userCommons) => { console.log("onSell called:",userCommons); }} 
 />
```

Then the story should start working properly.

**Review by pconrad — APPROVED** [review]
LGTM... though the storybook entry should be updated in a different PR (for 5 more points)

**Pull request merged**